### ISSUE TITLE
11. Nicely labeled graphics using ggplot

### ISSUE THREAD

**Issue by economicah** [economicah/publichousing23c#18]
color, line styles, tell a convincing story

**Comment by economicah** [comment]
LIZ: do you think we need to do this outside of the required graphics? my barplot uses ggplot and is nicely formatted

**Comment by eslawler** [comment]
Hmmm, not sure. It might be nice to have everything nicely labeled/colored regardless. I still need to add color to the scatter ggplots I made.

**Comment by economicah** [comment]
i agree! i will mark this as done-- we have it and will likely have even more.
